db/data/20210804083620_delete_zero_bps.rb

```diff
@@ -0,0 +1,16 @@
+class DeleteZeroBps < ActiveRecord::Migration[5.2]
+  def up
+    return unless CountryConfig.current[:abbreviation] == "ET"
+
+    zero_bps = BloodPressure.where(systolic: 0, diastolic: 0)
+
+    zero_bps.find_each do |blood_pressure|
+      blood_pressure.observation&.discard
```

Comment:
Can we just `delete` these records? We shouldn't really keep these around at all if we can help it.

Reply:
Hard deletion is a classic problem we haven't solved yet 😅  Deleting records on the server has no hook that notifies clients to delete them from device, so they will linger in our distributed network of data. Furthermore, anything that touches the records on client (in this case, only a data migration or something like that) will resync these deleted BPs back to the server, and the server will have no way of recognizing that they're deleted records, and create them again.